src/main/java/seedu/address/model/person/Email.java

```diff
@@ -44,29 +43,36 @@
      */
     public Email(String email) {
         requireNonNull(email);
-        checkArgument(isValidEmail(email), MESSAGE_CONSTRAINTS);
+        checkArgument(isValidEmailAddress(email), MESSAGE_CONSTRAINTS);
         value = email;
     }
 
     /**
-     * Returns if a given string is a valid email.
+     * Private constructor for Empty Email.
      */
-    public static boolean isValidEmail(String test) {
-        return isEmptyEmail(test) || test.matches(VALIDATION_REGEX);
+    private Email() {
+        value = "";
     }
 
     /**
-     * Returns true if the given string is the default empty email.
+     * Returns true if a given string is a valid email.
      */
-    public static boolean isEmptyEmail(String test) {
-        return test.equals(EMPTY_EMAIL);
+    public static boolean isValidEmailAddress(String test) {
+        return test.matches(VALIDATION_REGEX);
+    }
+
+    /**
+     * Returns true if the given email is the default empty email.
+     */
+    public boolean isEmpty() {
+        return this.equals(EMPTY_EMAIL);
     }
 
     /**
      * Returns true if the given email is the default email.
      */
-    public static boolean isDefaultEmail(Email test) {
-        return test.equals(DEFAULT_EMAIL);
+    public static boolean isValidEmail(Email test) {
```

Comment:
if we do this, then the one on JsonAdaptedPerson.java will fail, because now the email address can be an empty string. Basically isValidEmail checks whether the email is the one we defined, and isValidEmailAddress is for the user defined one.